icx/src/main.rs

```diff
@@ -567,14 +567,11 @@ async fn main() -> Result<()> {
                         print_idl_blob(&blob, &ArgType::Idl, &None)
                             .context("Failed to print request_status result")?;
                     }
-                    agent::RequestStatusResponse::Rejected {
-                        reject_code,
-                        reject_message,
-                    } => {
+                    agent::RequestStatusResponse::Rejected(replica_eror) => {
                         bail!(
-                            r#"The Replica returned an error: code {}, message: "{}""#,
-                            reject_code,
-                            reject_message
+                            r#"The Replica returned an error: code {:?}, message: "{}""#,
+                            replica_eror.reject_code,
+                            replica_eror.reject_message
```

Comment:
Can we have this include the error code?

Reply:
Yes, that makes sense. I have included it now.